src/kit/LogViewer/LogViewerSelect.tsx

```diff
@@ -65,6 +68,7 @@ const LogViewerSelect: React.FC<Props> = ({
     return Object.keys(selectOptions).reduce(
       (acc, key) => {
         const filterKey = key as keyof Filters;
+        if (filterKey === 'enableRegex') return acc;
```

Comment:
nit: it's weird having a boolean option handled in the same component that is otherwise almost all comboboxes. leads to weird special cases like here